microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -145,39 +143,38 @@ After you measured and verified that the animation isn't performing well, the ne
 
    ![The Main section.](../media/evaluate-performance-performance-main.msft.png)
 
-1. There's a lot of data in the recording.  To Zoom into a single event, click, hold, and drag your cursor over the **Overview**, which is the section that includes the **FPS**, **CPU**, and **NET** charts.  The **Main** section and **Summary** panel only display information for the selected portion of the recording.
+1. There are a lot of data in the recording.  To zoom into a portion of the recording, click, hold, and drag your cursor over the **Overview**, which is the section that includes the **CPU** and **NET** charts.  The **Main** section and **Summary** panel only display information for the selected portion of the recording.
 
-   ![Zoom into an event.](../media/evaluate-performance-performance-main-zoomed.msft.png)
+   ![Zoom into a section.](../media/evaluate-performance-performance-main-zoomed.msft.png)
 
-   Another way to zoom is to put focus on the **Main** section, click the background or an event, and press `W`, `A`, `S`, or `D`.
+   Another way to change the selected area is to put focus on the **Main** section, click the background or an event, and press `W` to zoom in, `A` to zoom out, `S` to move selection left, or `D` to move selection right.
 
-1. Focus on the red triangle in the top-right of the **Animation Frame Fired** event.  Whenever a red triangle is displayed, it's a warning that there may be an issue related to the event.
+1. Select a **Animation Frame Fired** event.  When a red triangle is displayed at the top right of an event, it's a warning that there may be an issue related to the event.
 
    The **Animation Frame Fired** event occurs whenever a [requestAnimationFrame() callback](https://developer.mozilla.org/docs/Web/API/window/requestAnimationFrame) is run.
 
-1. Click the **Animation Frame Fired** event.  The **Summary** panel now shows you information about that event.  Note the **Reveal** link.  After you click it, DevTools highlights the event that initiated the **Animation Frame Fired** event.  Also, focus on the **app.js:95** link.  After you click it, the relevant line in the source code is displayed.
+1. Click the **Animation Frame Fired** event.  The **Summary** panel now shows you information about that event.  Note the **Reveal** link.  After you click it, DevTools highlights the event that initiated the **Animation Frame Fired** event.  Also, focus on the **app.js:96** link.  After you click it, the relevant line in the source code is displayed.
```

Comment:
Possibly out of scope for this PR:
Unclear UI steps.  Are we directing user to click it?  If so, reformat/rewrite as standard separate action/result numbered steps.
I'd have to work through the rendered article to confirm that the steps need strengthening this way.

Reply:
For what it's worth, I went through these instructions with DevTools as I was revising the content, and this part made sense to me. This section is pointing out different places the developer might click to learn more information, though it may not always be necessary to do so, and is not necessary for the completion of this tutorial. I think it's OK to keep this section as is for now.

Reply:
ok.  pls click the Resolve Con. button.  I don't have the button.